python/plugins/db_manager/dlg_import_vector.py

```diff
@@ -132,6 +132,12 @@ def populateLayers(self):
             if layer is not None and layer.type() == QgsMapLayerType.VectorLayer:
                 self.cboInputLayer.addItem(layer.name(), layer.id())
 
+        # set the current index of the combo box to the active layer in the layer tree (if found in combo box)
+        if iface.activeLayer():
```

Comment:
```suggestion
        if iface is not None and iface.activeLayer():
```

(iface CAN be none in some circumstances, so let's play it safe!)